src/Components/Publishing/Article.tsx

```diff
@@ -68,7 +117,28 @@ export class Article extends React.Component<ArticleProps> {
     }
   }
 
+  shouldRenderSignUpCta = () => {
+    const { article, isLoggedIn, isMobile } = this.props
+
+    return isMobile && article.layout !== "series" && !isLoggedIn
```

Comment:
Worth double checking w @owendodd here if we should exclude from 'video' layouts as well -- they are also on a black background so it may not look right or need a different BG color.

Reply:
Good catch, thanks @eessex. Would be great to invert the banner color in these instances. So it would be a white banner with black text.